.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,51 @@
+{
+  "build": {
+    "dockerfile": "${localWorkspaceFolder}/.gitpod.Dockerfile"
+  },
+  "forwardPorts": ["26657:443", 1317, 8080, 7575, 4500],
+  "containerEnv": {
+    "GOMODCACHE": "/var/cache/go/modules",
+    "GOCACHE": "/var/cache/go/build",
```

Comment:
Maybe changing `GOMODCACHE` and `GOCACHE` to a path under `/workspaces` would fix the issue?

Reply:
I actually think I resolved the issue.

The problem was I was using the `localWorkspaceFolder` which is not available on code spaces. That said, the next push changes the location of the `.cache` directory to use the user's `$HOME/.cache/starport`, which I was trying to avoid but this is sadly a limitation of the devcontainer configuration file. It would probably be best to open a PR upstream to the plugin and see if they can expand on the lookup a bit or provide some XDG integration, but my hands are tied there